FoodTracker/MealTableViewController.swift

```diff
@@ -0,0 +1,139 @@
+                }
+
+                guard let indexPath = tableView.indexPath(for: selectedMealCell) else {
+                    fatalError("The selected cell is not being displayed by the table")
+                }
+
+                let selectedMeal = meals[indexPath.row]
+
+                mealDetailViewController.meal = selectedMeal
+
+        default:
+            fatalError("Unexpected Segue Identifier; \(String(describing: segue.identifier))")
+        }
+    }
+
+    @IBAction func unwindToMealList(sender: UIStoryboardSegue) {
+        if let sourceViewController = sender.source as? MealViewController,
+            let meal = sourceViewController.meal {
+
+            if let selectedIndexPath = tableView.indexPathForSelectedRow {
+                meals[selectedIndexPath.row] = meal
+
+                tableView.reloadData()
+            } else {
+                let newIndexPath = IndexPath(row: meals.count, section: 0)
+
+                meals.append(meal)
+
+                tableView.insertRows(at: [newIndexPath], with: .automatic)
+            }
+        }
+    }
+}
+
+private extension MealTableViewController {
+
+    func loadSampleMeals() {
+        let photo1 = UIImage(named: "meal1")
+        let photo2 = UIImage(named: "meal2")
+        let photo3 = UIImage(named: "meal3")
+
+        guard let meal1 = Meal(name: "Caprese Salad", photo: photo1, rating: 4) else {
+            fatalError("Unable to instantiate meal1")
+        }
+
+        guard let meal2 = Meal(name: "Chicken and Potatoes", photo: photo2, rating: 5) else {
+            fatalError("Unable to instantiate meal2")
+        }
+
+        guard let meal3 = Meal(name: "Pasta with Meatballs", photo: photo3, rating: 3) else {
+            fatalError("Unable to instantiate meal2")
+        }
+
+        meals += [meal1, meal2, meal3]
+    }
+}
+
+extension MealTableViewController {
+
+    override func tableView(_ tableView: UITableView, canEditRowAt indexPath: IndexPath) -> Bool {
+        return true
+    }
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return meals.count
+    }
+
+    override func tableView(_ tableView: UITableView,
+                            commit editingStyle: UITableViewCell.EditingStyle,
+                            forRowAt indexPath: IndexPath) {
+        if editingStyle == .delete {
+            meals.remove(at: indexPath.row)
+
+            tableView.deleteRows(at: [indexPath], with: .fade)
+        }
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cellIdentifier = "MealTableViewCell"
+
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: cellIdentifier,
+                                                       for: indexPath) as? MealTableViewCell
+            else {
```

Comment:
Too much newLines, is it because of Swiftlint line length?

If so, add .swiftlint.yml file to root of the project. This is what I usually use https://gist.github.com/Rep2/32d885ae1c3dc12159a3d5704e0590e9.

After that, remove newLines you think are too much.